tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "exactOptionalPropertyTypes": true,
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["ES2020", "dom"],
+    "module": "CommonJS",
+    "moduleResolution": "node",
+    "noEmit": true,
+    "noErrorTruncation": true,
+    "noUncheckedIndexedAccess": true,
+    "strict": true,
+    "target": "ES2017",
+    "noImplicitAny": false
+  },
+  "exclude": ["./dist/**/*", "node_modules"]
```

Comment:
Good call to add `node_modules` here — we should add this to the module template.

Reply:
I have opened a draft PR to update the configuration.

https://github.com/MetaMask/metamask-module-template/pull/181